post.md

```diff
@@ -0,0 +1,427 @@
+# Server Actions with User Confirmations in Next.js 13
+
+Next.js 13 has brought exciting new experimental React features to web development. The combination of server side rendered components and server actions have the potential to simplify web development while still giving us the full power of client side React when we need it.
+
+This article introduces the basics of server actions, and how to use them in combination with actions that require user confirmation. This is a common scenario for destructive actions such as deleting a record from a database.
+
+## Sample application
+
+This article presumes you're acquainted with setting up and running Next.js applications.
+
+To view the code referenced in this article, [checkout the repo](https://github.com/CodingZeal/nextjs-example-confirm-server-action).
+
+This article and accompanying example application utilize Next.js version 13.5.4. Bear in mind that server actions remain in the experimental phase and could evolve.
+
+## A Glimpse at Server Actions
+
+Next.js's server actions empower developers to execute server-side operations directly from the client without the hassle of wiring up HTTP client code. As the Next.js documentation succinctly puts it:
```

Comment:
Since this is a quote, should you also provide a link of exactly where you found this?